app/components/vnc-viewer.js

```diff
@@ -30,40 +29,32 @@ const VncViewerComponent = Component.extend({
   }),
 
   setupRFB() {
-    const rfb = this.get("rfb");
-    if (!isEmpty(rfb)) {
-      return;
-    }
     const canvasEl = this.element.getElementsByClassName("canvas")[0];
-    const that = this;
-    // eslint-disable-next-line no-undef
-    this.set("rfb", new RFB({
-      'target': canvasEl,
-      'encrypt': ENV.deviceFarmSsl,
-      'repeaterID': '',
-      'true_color': true,
-      'local_cursor': false,
-      'shared': true,
-      'view_only': false,
-
-      'onUpdateState'() {
-        setTimeout(that.set_ratio.bind(that), 500);
-        return true;
-      },
-
-      'onXvpInit'() {
-        return true;
+    const deviceToken = this.get("file.deviceToken");
+    const deviceFarmURL =
+      `wss://devicefarm2.appknox.com/websockify?token=${deviceToken}`
+
+    const rfb = new RFB(
+      canvasEl,
+      deviceFarmURL, {
+        'credentials': {
+            'password': '1234'
+        }
       }
-    })
     );
-
-    if (this.get('file.isReady')) {
-      return this.send("connect");
-    }
+    rfb.addEventListener('connect', () => {
+      const platform = this.get("file.project.platform");
+      if (platform === ENUMS.PLATFORM.IOS) {
+        rfb.scaleViewport = true;
```

Comment:
why not `scaleViewport = true` for both Android and iOS ?

Reply:
If we set `scaleViewport=true` for android, then the scales to half of the screen. This is only needed for iOS devices